config/300-containersource.yaml

```diff
@@ -0,0 +1 @@
+core/resources/containersource.yaml
```

Comment:
<!--whitespace-bot-->
Add trailing newline:
```suggestion
core/resources/containersource.yaml

```

Reply:
We should not bring this back into core. I recommend moving this to eventing-contrib as it is a deprecated component that really should not be used.